docker/buildkite/Dockerfile

```diff
@@ -10,19 +10,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
       curl \
       gettext-base \
       libyaml-dev \
-      python \
```

Comment:
this is already merged onto the branch and it's showing due to that github diffing issue